assets/app/scripts/controllers/build.js

```diff
@@ -99,8 +99,34 @@ angular.module('openshiftConsole')
           if (!$filter('isIncompleteBuild')(build)){
             delete $scope.buildConfigBuildsInProgress[buildConfigName][buildName];
           }
-        }        
+        }
       }));
+
+      (function initLogs() {
```

Comment:
Alternatively, i could do this:

```javascript
var initLogs = function() { /* logic */ }
initLogs(); // call it right away
angular.extend($scope, { initLogs: initLogs}); // then pass it to view
```

I think the first is a bit more idiomatic JS, but I could go either way.